leo_filters/include/complementary_filter.hpp

```diff
@@ -102,6 +112,22 @@ class ComplementaryFilter
   // Parameter whether to do adaptive gain or not.
   bool do_adaptive_gain_;
 
+  // Bias estimation thresholds:
+  // Parameter for threshold of the angular velocity
+  double angular_velocity_threshold_;
+
+  // Parameter for threshold of the acceleration
+  double acceleration_threshold_;
+
+  // Parameter for threshold of the delta angular velocity
+  double delta_angular_velocity_threshold_;
+
+  // Time required to be in steady state before bias estimation is performed.
+  double required_steady_time_;
+
+  std::chrono::steady_clock::time_point steady_start_time_;
+  bool in_steady_timer_;
```

Comment:
Small nitpick but I would rather use std::optional than additional bool variable.
```suggestion
  std::optional<std::chrono::steady_clock::time_point> steady_start_time_{std::nullopt};
```